tmp/lanelet2_extension/docs/lanelet2_format_extension.md

```diff
@@ -269,3 +269,26 @@ _An example:_
     <tag k="area" v="yes"/>
   </way>
 ```
+
+### No Drivable Lane
+
+A no drivable lane is a lanelet or more that are out of operation design domain (ODD), i.e., the vehicle **must not** drive autonomously in this/these lanelet/s.  
+A lanelet becomes no drivable by adding an optional tag under the relevant lanelet in the map file `<tag k="no_drivable_lane" v="yes"/>`.
+
+_An example:_
+
+```xml
+<relation id="2621">
+    <member type="way" role="left" ref="2593"/>
+    <member type="way" role="right" ref="2620"/>
+    <tag k="type" v="lanelet"/>
+    <tag k="subtype" v="road"/>
+    <tag k="speed_limit" v="30.00"/>
+    <tag k="location" v="urban"/>
+    <tag k="one_way" v="yes"/>
+    <tag k="participant:vehicle" v="yes"/>
+    <tag k="no_drivable_lane" v="yes"/>
+  </relation>
+```
+
+For more details about the `no_drivable_lane` concept and design, please refer to the [**_no-drivable-lane-design_**](../../../../../universe/autoware.universe/planning/behavior_velocity_planner/no-drivable-lane-design.md) document.
```

Comment:
Pre-commit is failing due to invalid link. 
I think it's due to other PR not being merged yet so I think we can merge this anyways since we know that it's going to be there.

Still, I think it's better to use the URL to github instead of local link:
```suggestion
For more details about the `no_drivable_lane` concept and design, please refer to the [**_no-drivable-lane-design_**](https://github.com/autowarefoundation/autoware.universe/tree/main/planning/behavior_velocity_planner/no-drivable-lane-design.md) document.
```

Reply:
Thanks @mitsudome-r san for the suggestion.
Will that solve the problem in the time that this file is not on main yet ?

Reply:
Fixed.
@mitsudome-r san : Please have a look